src/components/rubric-total-score.js

```diff
@@ -96,17 +96,23 @@ class RubricTotalScore extends composeMixins(
 				</d2l-button-subtle>
 			`;
 		}
-
 		return html`
-			<div class="out-of-container" tabindex="0">
+			<div class="out-of-container" tabindex="0" aria-labelledby="total-grouping-label">
+				<d2l-offscreen id="total-grouping-label" aria-hidden="true">
+					${this.localize('rubricEditorAccessibility:totalScoreOutOf', 
+						'rubricName', this.entity.name, 
+						'score', this.totalScore, 
+						'outOf', this.entity.outOf.toString())}
+				</d2l-offscreen>
```

Comment:
I think this will take care of the lint issues

```suggestion
				<d2l-offscreen id="total-grouping-label" aria-hidden="true">
					${this.localize(
						'rubricEditorAccessibility:totalScoreOutOf',
						'rubricName', this.entity.name,
						'score', this.totalScore,
						'outOf', this.entity.outOf.toString()
					)}
				</d2l-offscreen>
```

Reply:
NM doesn't take care of it :(

Reply:
I think you'll have to pull the localize call out of the HTML template.

```js
		const totalScoreOutOfText = this.localize(
			'rubricEditorAccessibility:totalScoreOutOf',
			'rubricName', this.entity.name,
			'score', this.totalScore,
			'outOf', this.entity.outOf.toString()
		);
		
		return html`
			<div class="out-of-container" tabindex="0" aria-labelledby="total-grouping-label">
				<d2l-offscreen id="total-grouping-label" aria-hidden="true">${totalScoreOutOfText}</d2l-offscreen>
```